rr_viz/src/rr_viz/widgets/mission_dashboard/mapping/slam_supervisor.py

```diff
@@ -256,6 +261,45 @@ def delete_map_slot(self):
                     print("failed calling slam delete map service")
                     print(trig_resp.message)
 
+    def save_map_image(self):
+        map_name = self.mapName.text()
+
+        if map_name !='':
+            if map_name.find('.'):
```

Comment:
IMO you should be able to just call `map_name.replace('.', '_')` with the same result and omitting this if check.